### DISCUSSION TITLE
Open a new tab upon connection to socket if tabs > 1

### DISCUSSION THREAD

**Discussion by bearcatsandor** [wezterm/wezterm#1170] [Q&A]
I love that wezterm has multiplexing built in, as it may help me accomplish something that I couldn't with alacritty + tmux.
Every time I launch 'wezterm connect unix' I'd like wezterm to open a new tab in the spawning wezterm window.   That way it appears as if i have 5 different terminals open on my desktop, but they are actually all the same stack, and that stack could be connected to in an ssh session.

Essencially this is the same as if I'd connected, and then hit super-shift-t.  My lua-fu is still weak. Help me oh wise ones!

**Comment by wez** [comment]
We're not geared up to do exactly that, but have you considered using `wezterm cli spawn` to launch a new tab in your existing gui window?

`wezterm cli --help` shows some of the other things you can do via the cli.
